### ISSUE TITLE
Unable to find subscription

### ISSUE THREAD

**Issue by jleach** [Microsoft/vscode-azure-account#60]
Error: Unable to find Subscription with name `my-subscription-id-here`

I signed up for a free trial over the weekend and I get this error. I'm not using a proxy, I've tried on several different internet connections. VSCode seems to be able to create storage account but can not deploy.

- VSCode 1.23.1
- Latest available extension version.

When I try and deploy a function I see error 51004 in the JSON of the Output window; I also see `"statusCode":404},` in there so something is not found.

I also tried with a Pay-As-You-Go and it still says it can not find my subscription. :(

**Comment by jleach** [comment]
Still and issue with VSCode 1.24.0.

**Comment by chrmarti** [comment]
@jleach Is there anything in the console? (Help > Developer Tools) /cc @EricJizbaMSFT

**Comment by jleach** [comment]
Just one INFO message "INFO no standard startup: not just one text editor"; nothing under Verbose, Warnings or Error.  When I switched from Free to Pay-As-You-Go VSCode did update the name of my subscription type internally (I now see Pay-As-You-Go in VSCode).

**Comment by jleach** [comment]
Ok, So rather than create a function and try to deploy it I created an Azure WebService (App Service) and deployed it. Now that my account is not completely empty - the error is *gone* under both Functions and App Service. I then stopped and deleted the serve - no error. Restarted VSCode and created a new Project/Workspace - again no error. Seems to have fixed it.

Looks like VSCode / Azure have a problem with fresh accounts that are completely empty; something must be created/updated when you deploy your first 'think'.

Hope this helps.
